lib/options.js

```diff
@@ -393,13 +400,33 @@ function refreshFilterPage() {
     // todo: gross hack, use templating framework
     var action = optionsGetAction(origin, originList);
     if(!action){ continue; }
+      ++cnt;
     printable = _addOriginHTML(origin, printable, action);
+      //console.log(printable);
   }
-  printable += "</div>"
+  printable.data += "</div>"
```

Comment:
I'm pretty sure printable is a string at this point, why are you adding to the `data` property of this string?